terraform/environments/analytical-platform-compute/s3-buckets.tf

```
@@ -22,6 +22,29 @@ module "mlflow_bucket" {
   tags = local.tags
 }
 
+data "aws_iam_policy_document" "s3_replication_policy" {
+  #checkov:skip=CKV_AWS_356:resource "*" being applied to replication iam role only
+  statement {
+    sid    = "Permissions on objects"
```

This _should_ be pascal case and maybe a bit more specific, e.g.

```suggestion
    sid    = "AllowLakeFormationPrincipalsReplication"
```